.github/workflows/ci.yml

```diff
@@ -0,0 +1,29 @@
+name: device-sdk-go
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ '*' ]
+
+jobs:
+  build:
+    name: go-vet-and-test
+    runs-on: ubuntu-18.04
+    steps:
+
+    - name: Set up Go 1.13.3
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1.13.3
+      id: go
+
+    - name: Check out code into the Go module directory
+      uses: actions/checkout@v2
+
+    - name: Get dependencies
```

Comment:
I recommend you to cache dependencies, like
https://github.com/mvdan/github-actions-golang#how-do-i-set-up-caching-between-builds

It uses `go.sum`'s hash (maybe md5) for a cache key. It enables us to drop the cache when dependencies are changed.

Reply:
The cache space is limited to 5GB and the following is GitHub Actions eviction policy.
https://help.github.com/en/actions/configuring-and-managing-workflows/caching-dependencies-to-speed-up-workflows#usage-limits-and-eviction-policy

Reply:
I see. thanks!